src/main/java/InputParser.java

```diff
@@ -1,4 +1,4 @@
-public class inputParser {
+public class InputParser {
 
     // Check if First Word is Keyword
     public boolean checkIfKeyWord(String Input) {
```

Comment:
if is boolean, hasKeyword/isKeyword would be a better naming? 

InputParser.isKeyword("abc").

Reply:
Yes. Indeed it would be. Thank you.